src/app2.js

```diff
@@ -0,0 +1,132 @@
+9. Gebruik deze functie de correcte zin, The capital is [city] and you can pay with [currency] and [currency]'s weer te geven.
+10. Maak een inputveld met zoek-knop op de pagina. In plaats van dat de data wordt opgehaald wanneer de pagina laadt, zorg je er nu voor dat de data over Nederland pas wordt opgehaald wanneer de gebruiker op ENTER of 'Zoek' drukt;
+11. Zorg ervoor dat de waarde uit het inputveld wordt gebruikt als dynamische waarde in jouw GET-request;
+12. Zorg ervoor dat de waarde van het input veld wordt leeggemaakt na elke zoekopdracht en dat er altijd maar één zoekresultaat op de pagina staat;
+13. Zorg ervoor dat als er naar een land wordt gezocht dat niet bestaat, er een foutmelding wordt getoond. Tip: als er een ongeldige API call wordt gemaakt, zal de response in het catch blok terecht komen.
+14. Zorg er ook voor dat wanneer er daarna een geldig verzoek wordt gedaan, de foutmelding weer verdwenen is.
+*/
+
+
+//Schrijf een asynchrone functie die, met behulp van Axios, een GET-request maakt naar het juiste endpoint voor nederland. Log de response in de console en bestudeer de data goed: hoe is het opgebouwd?
+import axios from 'axios';
+
+async function fetchCountry(nameCountry) {
+
+    const userWarning = document.getElementById('user-warning');
+    if (nameCountry !== "") {
+        try {
+        //Zorg ervoor dat de waarde uit het inputveld wordt gebruikt als dynamische waarde in jouw GET-request;
+        let output2 = await axios.get("https://restcountries.com/v2/name/" + nameCountry);
+
+        results(output2.data[0]);
+        userWarning.textContent = '';
+        }
+        catch (e) {
+            //Zorg ervoor dat als er naar een land wordt gezocht dat niet bestaat, er een foutmelding wordt getoond. Tip: als er een ongeldige API call wordt gemaakt, zal de response in het catch blok terecht komen.
+            userWarning.textContent = 'Het ingetypte land is niet correct, probeer weer';
+            console.log(e);
+        }
+    }
+
+}
+
+const userInput = document.getElementById('zoek-veld');
+
+userInput.addEventListener('keyup', checkUserInput)
+
+function checkUserInput(input) {
+
+
+    let buttonElement = document.getElementById('zoek-knop');
+    let buttonElement2 = document.getElementById('zoek-veld');
+
+
+    buttonElement.addEventListener('click', (e) => {
+        fetchCountry(input.target.value);
+        buttonElement2.value = "";
+    });
+
+    buttonElement2.addEventListener('keydown', (e) => {
+        if(e.code === 'Enter'){
+            document.getElementById("zoek-veld").click();
+             fetchCountry(input.target.value);
+             buttonElement2.value = "";
+        }
+    });
+}
+
+function results (output3) {
+    let currencys;
+   // Schrijf een functie die, ongeacht of er één of twee valuta's gebruikt worden, eenn string teruggeeft. Tip: dit kun je checken door bijvoorbeeld de informatie over panama op te halen.
+    function currencies (input) {
```

Comment:
Functies in functies is eigenlijk niet de bedoeling. Deze functie kan prima naast de `results` functie komen te staan